charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -21,6 +21,7 @@ data:
     http-server.http.enabled=false
     http-server.https.enabled=true
     http-server.https.port=8080
+    http-server.https.keystore.key=notnull
```

Comment:
@bentito we may need to also add this for the `internal-communication.https.keystore.key=notnull`. In any case, we should add a comment on why this change is needed for our future selves.

Reply:
done in 639a758f